package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "fluture-sanctuary-types",
+  "version": "0.0.0",
+  "description": "Fluture type definitions for Sanctuary",
+  "main": "index.js",
+  "scripts": {
+    "clean": "rimraf npm-debug.log .nyc_output coverage.lcov",
+    "lint": "eslint index.js test",
+    "release": "npm outdated --long && xyz --edit --repo git@github.com:fluture-js/fluture-sanctuary-types.git --tag 'X.Y.Z' --increment",
+    "test": "npm run lint && npm run test:unit",
+    "test:unit": "nyc mocha --ui bdd --reporter list --check-leaks --full-trace test/**.test.js",
+    "test:coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fluture-js/fluture-sanctuary-types.git"
+  },
+  "keywords": [
+    "fluture",
+    "sanctuary",
+    "types",
+    "typings",
+    "future",
+    "algebraic-data-types"
+  ],
+  "author": "Aldwin Vlasblom <aldwin.vlasblom@gmail.com> (https://github.com/Avaq)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/fluture-js/fluture-sanctuary-types/issues"
+  },
+  "homepage": "https://github.com/fluture-js/fluture-sanctuary-types#readme",
+  "dependencies": {
+    "sanctuary-type-identifiers": "^2.0.0"
+  },
+  "peerDependencies": {
+    "fluture": ">=5.0.0 <7.0.0",
+    "sanctuary-def": ">=0.8.0 <0.12.0"
```

Comment:
By looking at changes to the specific APIs being used by this module. Fluture 5.0.0 added `Future.Par`, which we rely on. And nothing else we're using changed since then. Sanctuary Def added the URL parameter to `BinaryType` in 0.8.0, and nothing changed since then either.